File: 006/solution.py
x=10
sum=0
for i in range(1,x+1):
  for j in range(1,x+1):
    if (i != j):
      sum += i*j

sq=0
for k in range(1,x+1):
  sq+=k

print "sum of squares is",(sq*sq-sum)
print "diff is",sum

File: 006/solution.cpp
#include<iostream>
using std::cout;
using std::endl;

const int GOAL = 100;

int main()
{
  unsigned long long difference = 0;
  for(int i=1;i<=GOAL;i++)
  {
    for(int j=1;j<=GOAL;j++)
    {
      if(i != j)
      {
        difference = difference + i*j;
      }
    }
  }
  unsigned long long sum= 0;
  
  for(int i=1;i<=GOAL;i++)
  {
    sum = sum + i;
  }

  cout << "The sum of all squares is " << (sum*sum - difference) << ", and the square of the sum is " << sum*sum << " which gives a difference of " << difference << "." << endl;
}
